packages/babel-plugin-redux-saga/package.json

```diff
@@ -12,6 +12,9 @@
   "scripts": {
     "test": "jest --env=node"
   },
+  "jest": {
```

Comment:
this shouldnt be needed because we already do this `"test": "jest --env=node"`

Reply:
sure, should be deleted. that's an old hack to fix tests locally